subsys/net/Kconfig.hostname

```diff
@@ -24,6 +24,18 @@ config NET_HOSTNAME_UNIQUE
 	  hostname. For example, zephyr00005e005357 could be the hostname
 	  if this setting is enabled.
 
+if NET_HOSTNAME_UNIQUE
+
+config NET_HOSTNAME_UNIQUE_UPDATE
+	bool "Update unique hostname"
+	default n
```

Comment:
You can remove the `default n` as the option is not set by default
Also, as this is just a lonely option, I would prefer `depends on NET_HOSTNAME_UNIQUE` instead of `if NET_HOSTNAME_UNIQUE`

Reply:
Thanks for your feedback. Added the requested changes.